airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -48,6 +54,101 @@
 )
 
 
+class DataProcCreateWorkflowBaseOperator(BaseOperator):
+    """Helper class for preparing configs from workflow template"""
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def _template_has_placement(self):
+        return True if "placement" in self.template else False
+
+    def _template_is_managed_cluster(self):
+
+        if self._template_has_placement():
+            return True if "managed_cluster" in self.template["placement"] else False
+        else:
+            # Placement is required field in the template
+            raise KeyError("Placement is missing")
+
+    def _has_valid_config(self):
+        if self._template_is_managed_cluster():
+            # cluster templates can be of 2 types, managed_cluster or cluster_selector.
+            return True if "config" in self.template["placement"]["managed_cluster"] else False
```

Comment:
```suggestion
            return "config" in self.template["placement"]["managed_cluster"]
```

There's a lot of places in this file which look like:
```python
return True if <condition> else False
```

Which can all be shortened to just 
```python
return <condition>
```